kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterContext.java

```diff
@@ -106,6 +108,21 @@ public interface FilterContext {
     <M extends ApiMessage> CompletionStage<M> sendRequest(RequestHeaderData header,
                                                           ApiMessage request);
 
+    /**
+     * Maps all of the given {@code topicIds} to the current corresponding topic names.
+     * @param topicIds topic ids to map to names
+     * @return CompletionStage for a TopicNameMapping. The TopicNameMapping is guaranteed to contain either
+     * a name or {@link org.apache.kafka.common.protocol.Errors} for each topic id requested. If a name or
+     * error cannot be determined for any topic id then this stage will be completed exceptionally with a
+     * {@link TopicNameMappingException}.
```

Comment:
I think failing the API call if a single Uuid is a really unfriendly interface.  I think it is better we signal an error for just the failing topic id.